### ISSUE TITLE
To be able to run pgxman on fedora based distro (centos, rocky, etc)

### ISSUE THREAD

**Issue by kesuskim** [pgxman/pgxman#148]
Hello, I was wondering whether can pgxman support fedora based distro like CentOS, Rocky Linux, RHEL, etc?

Currently it seems not possible.

```
[root@localhost ~]# curl -sfL https://install.pgx.sh | sh -

pgxman-install: need 'apt' (command not found)
```

**Comment by owenthereal** [comment]
Indeed, our current support is limited to apt-based Linux distributions, specifically Debian Bookworm and Ubuntu Jammy, as detailed in our [installation guide](https://docs.pgxman.com/installing_pgxman#linux-compatibility). We are working on extending our compatibility to include additional Linux distributions, such as those you've mentioned. Keep an eye out for updates!

**Comment by kesuskim** [comment]
Sorry for late response, bit hectic schedule 😅
Will look for an update on this! great project 👍
